npm/private/npm_translate_lock.bzl

```diff
@@ -133,32 +133,47 @@ def _gather_values_from_matching_names(keyed_lists, *names):
     return result
 
 def _get_npm_auth(rctx):
-    _NPM_TOKEN_KEY = "//registry.npmjs.org/:_authtoken"
-    token = None
+    _NPM_TOKEN_KEY = ":_authtoken"
```

Comment:
nice. this looks right. can you add a decent docstring to this function with some examples of what these values might look like in `.npmrc`? will help the next person who touches this code